docs/multicluster/architecture.md

```diff
@@ -34,75 +37,167 @@ given ClusterSet.
-member clusters. The above diagram depicts Antrea Multi-cluster resource export/import pipeline.
-
-Given two Services in the member clusters - `foo.ns.cluster1.local` and `foo.ns.cluster2.local`,
-multi-cluster Services may be generated by the following the resource export/import pipeline.
-
-* Administrators create resource ServiceExports `foo` in Namespace `ns` in each
-of the clusters.
-* The Exporters in member clusters `cluster1` and `cluster2` see ServiceExport `foo`, collect
-the associated Services and Endpoints, and create ResourceExports for them in the Common Area of the
-leader cluster.
-* The controller in the leader cluster sees ResourcesExports in the Common Area, computes multi-cluster
-Service `cluster1-ns-foo-service`, `cluster2-ns-foo-service` and associated Endpoints
-`cluster1-ns-foo-endpoints`, `cluster2-ns-foo-endpoints`.
-* The controller creates ResourcesImport enclosing multi-cluster Service `ns-foo-service` and Endpoints
-`ns-foo-endpoints` in the Common Area.
-* The Importer in each member cluster watches ResourceImports; decapsulates them and gets Service
-`ns/antrea-mc-foo`, Endpoints `ns/antrea-mc-foo`, and creates the resources and as well as a
-ServiceImport `ns/foo` locally if they don't exist or updates them if the resources have already
-been created by the Importer earlier.
-
-## Antrea Multi-cluster Service
-
-Antrea Multi-cluster Controller only supports Service of type ClusterIP at this moment. In
-order to support multi-cluster Service access between member clusters, Antrea requires member
-clusters' Pod IPs are reachable and no overlapping between all member clusters.
-
-When Antrea Multi-cluster Controller in member cluster watches ResourceImport's creation event
-in leader cluster, it will create multi-cluster Service, Endpoints and ServiceImport locally.
-The Service Ports definition will be the same as exported Services, the Endpoints will be Pod
-IPs from all member clusters. The new created Antrea Multi-cluster Service is just like a regular
-Kubernetes Service, so Pods in a member cluster can access the multi-cluster Service as usual without
-any extra setting.
+Antrea Multi-cluster Controller implements ClusterSet management and resource
+export/import in the ClusterSet. In either a leader or a member cluster, Antrea
+Multi-cluster Controller is deployed with a Deployment of a single replica, but
+it takes different responsibilities in leader and member clusters.
+
+### ClusterSet Establishment
+
+In a member cluster, Multi-cluster Controller watches and validates the ClusterSet
+and ClusterClaim CRs, and creates a MemberClusterAnnounce in the Common Area of the
+leader cluster to join the ClusterSet.
+
+In the leader cluster, Multi-cluster controller watches and validates the
+ClusterSet and Clusterclaim CRs, and initializes the ClusterSet. It also validates
+the MemberClusterAnnounce CR created by a member cluster and updates the member
+cluster status to the ClusterSet.
```

Comment:
Here is the main one I updated, we have a webhook for `MemberClusterAnnounce` in leader cluster, so I used `validates` here, and Admin is the one to add/update member clusters to ClusterSet, Controller will validate and update status only.